src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+plt.xlabel('Interatomic distance (pm)')
+plt.ylabel('Fusion rate (s$^{-\mathregular{1}}$)')
+
+plt.xlim(left=gamow_many_r["$r_2$"].min()/1e3)
+
+plt.ylim(ymin=1e-170)
+
+plt.tight_layout()
+
+plt.show()
+# plt.savefig("fusion-rate-distance.png", dpi=600)
+
+# # COMMENTED OUT CODE CAN BE USED TO REPRODUCE THE FORMAT OF PLOTS FROM https://doi.org/10.1088/1367-2630/ad091c
+# fig, ax = plt.subplots(figsize=(8, 6))
+# plt.yticks([1e-40,1e-60,1e-80, 1e-100, 1e-120, 1e-140, 1e-160], [f"{tick:.0E}" for tick in y_ticks])
+# plt.xlabel('Interatomic distance (pm)', fontsize=25, fontname='Arial')
+# plt.ylabel('Fusion rate (s$^{-\mathregular{1}}$)', fontsize=25, fontname='Arial')
+# plt.xticks(fontsize=25, fontname='Arial')
+# plt.yticks(fontsize=25, fontname='Arial')
+# plt.gca().invert_xaxis()
+# ax = plt.gca()
+# ax.spines['top'].set_visible(False)
+# ax.spines['right'].set_visible(False)
+```
+
+From the above plot, we can see that, in the absence of electron screening, the fusion rate for deuterium in palladium O-sites (at 250 pm) would be about 82 orders of magnitude smaller than for $\rm D_2$ in a gas.
+
+Let's now see how much of an effect electron screening can have.
+
+
+## Screened fusion rates
+
+
+We know that $\rm D_2$ can get absorbed into metals like Palladium. The abundance of electrons inside a metal invites us to consider more deeply the effect of electron screening on fusion rates. 
+
+Simplistically, electron screening reduces the positive electrostatic repulsion between nuclei and results in a reduced interatomic distance and hence higher fusion rates. Most theoretical descriptions of screening in metals have been developed to attempt to explain anomalously high fusion rates in low energy beam-target experiments (see e.g. [Huke et.al](http://dx.doi.org/10.1103/PhysRevC.78.015803)). These models have had some success, but there are still discrepancies with experiments and a rigorous theory of the extremely low energy regime has yet to be agreed upon.
+
+We will construct a screening model that incorporates aspects of the molecular potential we've used for $\rm D_2$ in addition to the standard screening ideas from the beam-target scenario.
+
+The simplest model of screening for the beam-target scenario takes the form of a screened Coulomb potential:
+
+$$V_{\rm screened} = \frac{e^2}{4\pi\epsilon_0r} e^{-\frac{r}{\lambda}}$$
+
+$\lambda$ is known as the screening length and gives a measure of the size of an electron cloud surrounding a nucleus. The screening length can be expressed as terms of a screening energy, $U_e$, via:
+
+$$U_e = \frac{e^2}{4\pi\varepsilon_0 \lambda}$$
+
+which has an upper limit in metals of about 300 eV.
+
+While it is common to simplify $V_{\rm screened}$ by performing a taylor expansion of the exponential, this is only justified for high beam energies $E \gg U_e$ where the classical tuning points for tunelling satisfy $r \ll \lambda$. For very low energies, the full exponential must be used.
```

Comment:
here, I would capitalized Taylor in Taylor expansion (just like we capitalized Gamow in Gamow model)